less/project/columns.less

```diff
@@ -16,8 +16,8 @@
 // min small viewport takes up 8 out of 12 columns
 // min medium viewport takes up 10 columns
 // if undefined element defaults to 100% width
-.make-columns(xs, 0, 12);
+.make-columns(xs, 0, 12); // deprecated: backward compatibility only
```

Comment:
I think maybe XS should stay anyway as its deprecated in the sense that the defaults breakpoint vals make it not needed but I guess it's possible it may be useful in some cases if someone sets the small breakpoint above zero?